src/quilla/hookspecs/reports.py

```diff
@@ -0,0 +1,19 @@
+'''
+Hooks related to outputs and reports
+'''
+
+from quilla.ctx import Context
+from quilla.hookspecs import hookspec
+from quilla.reports.report_summary import ReportSummary
+
+
+@hookspec
+def quilla_postvalidate(ctx: Context, reports: ReportSummary):
```

Comment:
Post validation could have more than just output and report. Why not just name it in a more generic way?

Reply:
Post-validation the reports are already generated and all the validations are done, I might rename this as "quilla_post_test" instead? Since this specific hook is called after "validate_all()" is called, the only thing left to do is do the reporting output

Reply:
Added #21  to address this later